routes/index.js

```diff
@@ -28,7 +22,7 @@ router.get('/', function(req, res) {
 		}
 	}], function(err, result){
 		temparray = [];
-		endresult = [];
+		var endresult = [];
 		if (result){
 			temparray.push(result.slice(0,3))
```

Comment:
whats going on here? we slice the array and then only get the first result AND then loop through when it's length has to be 1? im confused